ports/onednn/portfile.cmake

```
@@ -1,8 +1,8 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO oneapi-src/oneDNN
-    REF 58be3660fb57c4c4a3d306730e849237d1271572 #2.6.1
-    SHA512 8fc51655d0ecdb5b137d04c6b3ee01959d9505e0d7ee74cab872447b5873f214d38db55edf4040a56d4455557ec2c0707097337c6e276dbaf1c9c6e48b95a820
+    REF 3fef24ec08eb89a2c4be4425162c0651af7f1de3 #3.0
```

```suggestion
    REF "v${VERSION}"
```
Here suggest use `"v${VERSION}"`, it could avoid change in next update.